common/journal_entries/dvg_lotharingia.txt

```diff
@@ -0,0 +1,329 @@
+je_lotharian_house_of_cards = {
+	icon = "gfx/interface/icons/event_icons/event_scales.dds"
+
+	is_shown_when_inactive = {
+		exists = c:LOT
+		c:LOT = ROOT
+		NOT = { has_variable = meiji_restoration_complete }

```

Comment:
Why are you checking this variable?

Reply:
leftover from shogunate JE, updated to "house_of_cards_complete" variable